pelutils/__init__.py

```diff
@@ -119,6 +121,34 @@ def split_path(path: str) -> list[str]:
     return os.path.normpath(path).split(os.sep)
 
 
+class jsonl:
```

Comment:
Det her skal da være et modul?

Reply:
Principielt ja, men tænkte, at der er for lidt kode til, at et modul giver mening? Men kan godt ændre det for din skyld :*